src/util/util_tests.py

```diff
@@ -1,9 +1,12 @@
-from np_utils import intersect_2d 
+from np_utils import intersect_2d, orientation_2d
 import numpy as np
+from dataclasses import dataclass
 
 """
 Unit tests for intersect_2d
 Implicitly check orientation_2d
+
+Delete this file later
```

Comment:
you should just put the unit tests in the unit testing folder (and then we don't have to delete it either)

Reply:
didn't know that existed lol I'll do that